AppSandboxFileAccess/Classes/AppSandboxFileAccess.h

```diff
@@ -36,6 +36,20 @@
 #import <Foundation/Foundation.h>
 @import AppKit;
 
+#pragma mark -
+#pragma mark AppSandboxFileAccessProtocol
+
+@protocol AppSandboxFileAccessProtocol<NSObject>
+
```

Comment:
If you add in `@required` on this line, the compiler should ensure anything implementing the protocol has the functions listed below it. Then you don't need to do the conforming check in `setBookmarkPersistanceDelegate`. 😄